packages/dispatcher-converter/src/converter/singleFilesConverter.js

```diff
@@ -199,6 +198,23 @@ class SingleFilesConverter {
             "Single File Converter: Creating farm file from dispatcher.any"
         );
 
+        fs.writeFileSync(path.join(Constants.TARGET_DISPATCHER_SRC_FOLDER,
+                        "tempFile.txt"),"");
+        for(let i=0;i<fileContentsArray.length;i++){
```

Comment:
Please fix the formatting.
Also can you add comments explaining what is being done here? It becomes easier to understand for someone else who might be working on the code in future. 
Can you please create a separate PR in future for adding comments, explaining complex operations throughout the dispatcher converter code?

Reply:
@prateeks you can yarn run lint  before committing changes to see formatting errors and then you can run npm run lint -- --fix to automatically fix those errors